prow/plugins.yaml

```diff
@@ -235,6 +235,10 @@ repo_milestone:
     maintainers_id: 3169231
     maintainers_team: community-milestone-maintainers
     maintainers_friendly_name: Community Milestone Maintainers
+  kubernetes/website:
+    maintainers_id: 2953475
```

Comment:
The `maintainers_id` is automatically assigned by GitHub once the team is created and you can find that by `curl -H "Authorization: token <token>" "https://api.github.com/orgs/<org-name>/teams?page=N"`...I'm curious what this number corresponds to? :thinking:

Reply:
@nikhita 

> I'm curious what this number corresponds to?

It's a placeholder for the ID that will be created when https://github.com/kubernetes/org/pull/644 merges.

Reply:
@nikhita 👋 I updated this after https://github.com/kubernetes/org/pull/644 merged. This PR should be ready to go.

```json
  {
    "name": "website-milestone-maintainers",
    "id": 3175912,
    "node_id": "MDQ6VGVhbTMxNzU5MTI=",
    "slug": "website-milestone-maintainers",
    "description": "Contributors who can use `/milestone` in the website repo.",
    "privacy": "closed",
    "url": "https://api.github.com/teams/3175912",
    "html_url": "https://github.com/orgs/kubernetes/teams/website-milestone-maintainers",
    "members_url": "https://api.github.com/teams/3175912/members{/member}",
    "repositories_url": "https://api.github.com/teams/3175912/repos",
    "permission": "pull"
  }
```